File: AreaTriangulo.cpp
#include <iostream>
int main(){
  printf("Ejercicio 01: Area de un triangulo \n");
  //declarar variables
  int b,h,area;
  std::cout<<"ingrese base: ";
  std::cin>>b;
  std::cout<<"ingrese altura: ";
  std::cin>>h;
  //operacion
  area=b*h/2;
  //salida
  std::cout<<"el Area del triangulo es "<<area;
  return 0;
}

File: AreaTriangulo.go
package main
import "fmt"
func main(){
  //declaracion de variables 
  fmt.Println("Ejercicio 01: Area del Triangulo")
  var b int
  var h int
  var area float64
  //datos de lectura
  fmt.Println("Ingrese Base:  ")
  fmt.Scanln(&b) 
  fmt.Println("Ingrese Altura:") 
  fmt.Scanln(&h) 
  //Proceso 
  area=float64((b*h)/2) 
  //Datos de salida 
  fmt.Println("El area es: ", area)
}